app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +27,6 @@
         </activity>
         <meta-data
             android:name="com.stripe.android.API_KEY"
-            android:value="${STRIPE_API_TOKEN}"/>
+            android:value="${STRIPE_API_TOKEN}" />
```

Comment:
Extra

Reply:
Since I've added RECORD_AUDIO permission in manifest, everyline went down by one.
I've checked it in my IDE.
In mine, last line is line30.
No extra space is there.

Reply:
Removed.